### DISCUSSION TITLE
Welcome to ExpertSDR3-SUGGESTIONS Discussions!

### DISCUSSION THREAD

**Discussion by ExpertSDR3** [ExpertSDR3/ExpertSDR3-SUGGESTIONS#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 1972igor** [comment]
ESDR3 v 0.10.0
Допускаю, что можно использовать «Микрофонный метр» для мониторинга каждого модуля «ПРОК», но думаю, что этот индикатор устанавливается на входе модулятора, после внутреннего АРУ передатчика, АРУ передатчика работает после 0дБ! И мы не увидим, какой модуль обработки имеет уровень выше 0 дБ!! Но сегодня индикатор Mic meter дает нам неверную информацию о входном уровне, а именно он не работает линейно на нижних проходах!!! нужны точные измерители уровня для каждого модуля / этапа в меню proc.
OdB сигнала на микрофонном входе должен показывать 0 дБ на метрах, он также должен быть линейным и корректным считываться на более низких уровнях. Измерение для каждого этапа обработки поможет пользователю поддерживать правильный уровень усиления и удерживать от чрезмерного или недостаточного вождения P.A в радио 
![image](https://user-images.githubusercontent.com/93991940/144989712-e4791ff7-2355-4bb4-83db-1695c22e3f4b.png)
![image](https://user-images.githubusercontent.com/93991940/144989761-e76129ca-2e8b-44f2-8314-002fc0b8679c.png)
![image](https://user-images.githubusercontent.com/93991940/144989803-b73c3058-c0e7-43db-a120-0103d4b7f6e0.png)
![image](https://user-images.githubusercontent.com/93991940/144989829-6e5c9817-0311-4370-8e67-ec9b165afbaa.png)
![image](https://user-images.githubusercontent.com/93991940/144989874-fdb65c0b-d139-4d0e-8137-f7c1b6a3cff9.png)
![image](https://user-images.githubusercontent.com/93991940/144989954-e26d3cfd-781c-4041-a9ad-694a0b3b1dc2.png)
![image](https://user-images.githubusercontent.com/93991940/144989993-57200a86-7694-491f-8f4d-4f3878a5ef4b.png)
![image](https://user-images.githubusercontent.com/93991940/144990035-8869c4e6-9b59-4adf-8044-31b99905c5ef.png)
![image](https://user-images.githubusercontent.com/93991940/144990071-99b19d2e-cd96-490f-ad86-312f09ade02e.png)

**Comment by 1972igor** [comment]
Вариант Джона понравился больше, не нужно бегать и переключать счетчик в разные точки, все сразу видно, что и где перегружено!!! И дизайн прорисован лучше, чем мой !!! Браво Джон!!! Команде Expert Electronics остается быстро внедрить его, а не обещать это сделать!!! :стена: :стена: :стена:
P/S Хотя индикатор после модуля постоянного тока и рота не нужны, эти модули не вносят изменений в уровень сигнала!!! :write: :write:
Я уже измерил его!!!
